### ISSUE TITLE
Name Change's Effect on Friends List

### ISSUE THREAD

**Issue by Pyromanical** [Wynncraft/Issues#7360]
As you know, when someone changes their name, and logs in, it updates on your friends list.  However, one of my friends who hadn't played in a number of months, and had changed their name logged it.  They joined with the name Synthoo, then left, and changed their name to _xZephyr, then back to Synthoo, and finally rejoined as SoPerfect.   However, when they rejoined they weren't on my friends list or shown as online.  I had to re-add them with their new name.  Changing names multiple times seems to not update your friends list.

tl;dr: If a friend changes their name more then once while offline, your friends list won't update.
![img_1254](https://cloud.githubusercontent.com/assets/21978993/22850536/7b844d96-efd0-11e6-9a55-736986442051.JPG)
Pic of friends list with both names on it (ignore bad quality, I can't screenshot with tab open)
Stupid sideways picture

**Comment by DebugMC** [comment]
Wrong, I have changed my name over 7 times and ppl still tell me they saw me as Mario1230X (That was like wynn 1.5)

**Comment by Moe-Ronickah** [comment]
@Pyromanical , try going through your friends list, and see if the old names are still there.  If they are, remove them.  It won't fix the current bug, but it might help in the future with friends that keep changing their names.

**Comment by LordHelmchen98** [comment]
It has been more than 30 days without a response to this issue, or there have been no further reports of this issue, or it is a Minecraft mechanic and not related to WynnCraft. If you are still experiencing this issue, you can open a new bug report.
